### ISSUE TITLE
Flatpack?

### ISSUE THREAD

**Issue by Nico7as** [Timmmm/robocut#5]
Hi,

Could Robocut be packaged and published as a flatpack on Flathub, so it becomes distro-agnostic?

Nicolas

**Comment by Timmmm** [comment]
Good idea. I don't actually have a desktop Linux system to test on at the moment though (or really any free time!) but I would definitely welcome this.

(Note: if anyone enterprising does add this, do it on the `develop` branch which is way ahead of `master`. It's more or less ready to merge into `master` and do a release but I haven't got around to fully testing it yet.)
